conseil-api/src/main/scala/tech/cryptonomic/conseil/api/ConseilMainOutput.scala

```diff
@@ -56,7 +56,8 @@ trait ConseilMainOutput {
       .groupBy(_.platform)
       .map {
         case (platform, configuration) =>
-          val networks = configuration.map(_.network).mkString("\n  - ", "\n  - ", "\n")
+          val networks =
+            configuration.map(c => s"${c.network} (enabled: ${c.enabled})").mkString("\n  - ", "\n  - ", "\n")
```

Comment:
Will it be less "noisy" if we simply print a flag when a network is disabled?

would look like

```
 - mainnet
 - netflix (disabled)
 - fakenet
```?

Reply:
Well, in a sea of networks then probably this proposition would work better, but considering there are mostly 2 by blockchain (if even), then this change from readability perspective is rather cosmetic. Nonetheless, I've change that to what you proposed, hope it will looks nice now :)